app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
toán tử `||=` có tác dụng gì?

Reply:
`@current_user` là giá trị bên trái nếu nó true, không thì gán băng với vế phải

Reply:
chưa chuẩn lắm em nhé
nếu `@current_user` nil thì tiến hành tìm kiếm để lấy giá trị, còn ko thì lấy giá trị chính nó